pymc_experimental/tests/custom_rjmcmc_sketch.py

```diff
@@ -0,0 +1,333 @@
+from step_methods.rjmcmc import *
+
+import functools
+import itertools
+
+from scipy.linalg import expm
+import scipy
+import numpy as np
+
+import pymc as pm
+from pymc.aesaraf import compile_pymc
+
+import dill as pickle
+import arviz as az
+
+
+# Since pymc4 seems to be using Aesara (their own version of thano) instead of the old theano
+# import aesara_theano_fallback.tensor as tt
+import aesara_theano_fallback.tensor as tt
```

Comment:
Since you're importing `pymc` above, this is already tailored to v4.
You can just do this:
```suggestion
import aesara.tensor as at
```
(`at` is the convention. Just find & replace.)